swarm/swarm.go

```diff
@@ -229,8 +229,11 @@ func NewSwarm(ctx *node.ServiceContext, backend chequebook.Backend, config *api.
 	self.bzz = network.NewBzz(bzzconfig, to, stateStore, stream.Spec, self.streamer.Run)
 
 	// Pss = postal service over swarm (devp2p over bzz)
-	pssparams := pss.NewPssParams(self.privateKey)
-	self.ps = pss.NewPss(to, pssparams)
+	config.Pss.WithPrivateKey(self.privateKey)
```

Comment:
this is a tad bit strange IMO. `WithPrivateKey` returns a value, and although the value within `config.Pss` will mutate, I'd rather explicitly use an equals here:
```
config.Pss = config.Pss.WithPrivateKey(self.privateKey)
```

fix on your own judgment, otherwise LGTM